index.client.js

```diff
@@ -56,17 +57,27 @@ ReactI13nGoogleAnalytics.prototype.getPlugin = function () {
              * @param {Function} calback callback function
              */
             pageview: function (payload, callback) {
+                var args = {},
+                    argsKeys = ['location', 'page', 'title'];
+
+                argsKeys.forEach(function (prop) {
```

Comment:
that's about a good coding manner, so that we won't see `(anonymous function)` in console